### PR TITLE
9 personal page

### PR DESCRIPTION
**Pull request by emarc99**
## Description

Build personal builder's page.

image

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

Your ENS/address: emarc.eth

### CODE REVIEW

**Review by phipsae — CHANGES_REQUESTED** [review]
Thank you so much for submitting your PR for your personal page!

Just a quick note: please make sure that only the relevant files are added or changed. In your case that should be only `backgroundPattern.tsx`, `page.tsx`, and your picture. Any unrelated files should stay untouched. Once you've made these adjustments, I’ll review your page.

**Comment by emarc99** [comment]
I have applied requested changes.

File: packages/nextjs/app/builders/0x11Bc455A4f64F800f9Ceb37614f02b213f98591d/page.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      label: "Twitter",
+      icon: <FaXTwitter size={24} />,
+    },
+    {
+      href: "https://www.linkedin.com/in/emmanuel-aroso-002783169/",
+      label: "LinkedIn",
+      icon: <FaLinkedin size={24} />,
+    },
+    {
+      href: "https://discord.com/emarc99", // Can be your Discord username or server invite
```
Comment (phipsae) [inline packages/nextjs/app/builders/0x11Bc455A4f64F800f9Ceb37614f02b213f98591d/page.tsx:25]: Somehow the discord link doesn't work.

File: packages/nextjs/app/builders/0x11Bc455A4f64F800f9Ceb37614f02b213f98591d/page.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import Image from "next/image";
+import { backgroundPattern } from "./backgroundPattern";
+import { NextPage } from "next";
+import { FaDiscord, FaGithub, FaLinkedin, FaTelegram } from "react-icons/fa";
+import { FaXTwitter } from "react-icons/fa6";
+
+const EmarcProfile: NextPage = () => {
+  const socialLinks = [
```
Comment (phipsae) [inline packages/nextjs/app/builders/0x11Bc455A4f64F800f9Ceb37614f02b213f98591d/page.tsx:8]: You could have to `socialLinks` array outside of the component. It prevents unnecessary recreation on each render.

File: packages/nextjs/app/builders/0x11Bc455A4f64F800f9Ceb37614f02b213f98591d/page.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+          </div>
+        </figure>
+        <div className="card-body items-center text-center">
+          <h2 className="card-title text-2xl font-bold  bg-clip-text ">Aroso Emmanuel</h2>
+          <p className="text-sm mt-2 leading-relaxed max-w-md opacity-90 hover:opacity-100 transition-opacity duration-300">
+            A researcher with computer engineering background, currently exploring the web3 space for new research
+            opportunities and contributing to interesting projects.
+          </p>
+          <h2 className="fmt-2 leading-relaxed max-w-md opacity-90 hover:opacity-100 transition-opacity duration-300">
+            ENS/Wallet Address:<span className="font-bold text-2xl"> emarc.eth</span>
```
Comment (phipsae) [inline packages/nextjs/app/builders/0x11Bc455A4f64F800f9Ceb37614f02b213f98591d/page.tsx:59]: Here you could use the SE2 component `<Address>`

**Review by phipsae — CHANGES_REQUESTED** [review]
Thanks for your nice personal page, like your background! 

Could you please pull the latest changes from the main branch and merge them into your branch? So you'll always ensure compatibility with the latest updates to the main branch.

Also, take a look at the little comments below. 
Once those are addressed, we’ll be ready to merge.

**Comment by emarc99** [comment]
Page is ready, thank you.

**Comment by phipsae** [comment]
Thanks a lot for the page! Time to merge.

**Pull request merged**